package.json

```diff
@@ -7,14 +7,10 @@
     "express": "^4.17.1",
     "globby": "^10.0.1",
     "highlight.js": "^9.15.10",
-    "hint": "^5.2.3",
```

Comment:
Need to install `hint` somewhere to test staging or we could use `npx hint https://staging`